server/plugin/command.go

```diff
@@ -320,14 +329,14 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 
 	ctx := context.Background()
 	githubClient := p.githubConnectUser(ctx, userInfo)
-
+	username, _ := p.API.GetUser(args.UserId)
```

Comment:
Should we check the error here?

```suggestion
	user, _ := p.API.GetUser(args.UserId)
```